docs/pages/guides/authentication.mdx

```diff
@@ -1782,6 +1783,73 @@ const { accessToken } = await AuthSession.exchangeCodeAsync(
 </Box>
 {/* End Strava */}
 
+<Box
+  name="Strivacity"
+  createUrl="https://www.strivacity.com/learn-support/developer-hub"
+  image={ASSETS.strivacity}
+>
+
+| Website                     | Provider  | PKCE      | Auto Discovery |
+| --------------------------- | --------- | --------- | -------------- |
+| [Get Your Config][c-strivacity] | OpenID | Supported | Available  |
+
+[c-strivacity]: https://docs.strivacity.com/docs/setting-up-your-instance
+
+{/* prettier-ignore */}
+```tsx Strivacity Auth Example
+import * as React from 'react';
+import * as WebBrowser from 'expo-web-browser';
+import { makeRedirectUri, useAuthRequest } from 'expo-auth-session';
+import { Button } from 'react-native';
+
+/* @info <strong>Web only:</strong> This method should be invoked on the page that the auth popup gets redirected to on web, it'll ensure that authentication is completed properly. On native this does nothing. */
```

Comment:
```suggestion
/* @info <strong>Web only:</strong> This method should be invoked on the page that the auth popup gets redirected to on web. It will ensure that authentication is completed properly. On native platforms, this does nothing. */
```